### PR TITLE
Organization

### PR DESCRIPTION
**Pull request by egbertn**

### CODE REVIEW

File: EventbriteApiV3/Attendees/AttendeeSearchEventbriteRequest.cs [hunk 1]
```diff
@@ -9,13 +10,19 @@ public class AttendeeSearchEventbriteRequest : EventbriteRequestBase
...
-           
+
         }
 
         public AttendeeSearchApiResponse GetResponse()
         {
             var response = GetJsonResponse();
 
+            return JsonConvert.DeserializeObject<AttendeeSearchApiResponse>(response);
+        }
```
Comment (DTONeill) [inline EventbriteApiV3/Attendees/AttendeeSearchEventbriteRequest.cs]: Put a new line between methods please.

File: EventbriteApiV3/EventbriteContext.cs [hunk 2]
```diff
@@ -19,10 +21,24 @@ public EventsSearchApiResponse GetEvents(BaseSearchCriterias searchCriterias)
         {
             return new EventSearchEventbriteRequest(this, searchCriterias).GetResponse();
         }
+        public  Task< EventsSearchApiResponse> GetEventsAsync(BaseSearchCriterias searchCriterias)
+        {
+            return (new EventSearchEventbriteRequest(this, searchCriterias)).GetResponseAsync();
+        }
```
Comment (DTONeill) [inline EventbriteApiV3/EventbriteContext.cs:28]: New line here.

File: EventbriteApiV3/EventbriteContext.cs [hunk 3]
```diff
@@ -19,10 +21,24 @@ public EventsSearchApiResponse GetEvents(BaseSearchCriterias searchCriterias)
...
+            return (new EventsOrganizationRequest(this, organisationId, searchCriterias)).GetResponseAsync();
 
+        }
         public AttendeeSearchApiResponse GetAttendees(double eventId, BaseSearchCriterias searchCriterias)
         {
             return new AttendeeSearchEventbriteRequest(this, eventId, searchCriterias).GetResponse();
         }
+
+
+		public Task< AttendeeSearchApiResponse> GetAttendeesAsync(double eventId, BaseSearchCriterias searchCriterias)
```
Comment (DTONeill) [inline EventbriteApiV3/EventbriteContext.cs]: Remove the tab before "public"

File: EventbriteApiV3/Events/EventSearchEventbriteRequest.cs [hunk 4]
```diff
@@ -9,13 +10,19 @@ public class EventSearchEventbriteRequest : EventbriteRequestBase
...
-           
+
         }
 
         public EventsSearchApiResponse GetResponse()
         {
             var response = GetJsonResponse();
 
+            return JsonConvert.DeserializeObject<EventsSearchApiResponse>(response);
+        }
```
Comment (DTONeill) [inline EventbriteApiV3/Events/EventSearchEventbriteRequest.cs:20]: new line here

File: EventbriteApiV3/EventbriteRequestBase.cs [hunk 5]
```diff
@@ -49,5 +50,23 @@ protected string GetJsonResponse()
...
+            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(Url);
+            request.Headers.Add("authorization", $"Bearer {Context.AppKey}");
+            var response = await request.GetResponseAsync();
+
+            using (Stream stream = response.GetResponseStream())
+            using (StreamReader sr = new StreamReader(stream))
+            {
+                return await sr.ReadToEndAsync();
+            }
+        }
```
Comment (DTONeill) [inline EventbriteApiV3/EventbriteRequestBase.cs]: new line here

**Review by DTONeill — CHANGES_REQUESTED** [review]

**Review by DTONeill — CHANGES_REQUESTED** [review]
Since the conflict resolution I don't see anymore EventsOrganizationRequest or the method getEventsByOrganisation.
Look at the "files changed tab"

**Comment by egbertn** [comment]
> 
> 
> Since the conflict resolution I don't see anymore EventsOrganizationRequest or the method getEventsByOrganisation.
> Look at the "files changed tab"

Cannot reproduce :( 
Done everything, git merge upstream etc. Is it an idea that you close the PR's, and I recreate one new PR?

**Comment by DTONeill** [comment]
Yes i think it will be better if you create a new branch, copy paste the code and create a new pull-request.

**Pull request closed without merging**